# Study 948 — Capture Ratio 📉

**Do income funds really "give up a little upside to avoid a lot of downside"?**

That sentence is on every covered-call and high-dividend fact sheet, and it is a claim you
can *measure*. Take a fund's monthly returns, split the months by whether its benchmark was
up or down, and ask what share of each it kept:

* **up-capture** — the share of the benchmark's good months the fund kept,
* **down-capture** — the share of the bad months it took,
* **capture spread** — up minus down. **Positive means the promise is true.**

We score **14 US income ETFs** — nine option-writers and five dividend funds as
a no-options control — against the index each is marketed against (SPY / QQQ / IWM), on
monthly **total** returns (they pay most of their return out as distributions), cash leg
BIL, 2007-06-30 → 2026-06-30 (327 month-ends), as-of 2026-06-30.

*Real numbers below are the frozen headline (`docs/results.md`, fingerprint
`01c183fa5741`). The live cells run the offline synthetic checks and say so.*


## 1. The promise, and the one number that tests it

A fund that keeps **70%** of the good months and only **40%** of the bad ones is doing something genuinely clever: it has bent the payoff in your favour. A fund that keeps 70% of the good months and 70% of the bad ones has done nothing you could not do yourself by holding 70% of the index and putting the rest in the bank.

So the honest scorecard is not the two numbers. It is the **gap** between them.

## 2. What the tape says

Fourteen funds. Up-capture, down-capture, the spread, and a 95% confidence interval on that spread. Read the last two columns.

In [1]:
R_SCORE = {'QYLD': ('QQQ', 150, 0.497, 0.559, -0.062, -0.21, 0.11, 0.23, -0.35, -2.62), 'JEPQ': ('QQQ', 49, 0.712, 0.689, 0.023, -0.13, 0.24, 0.64, -0.44, -5.58), 'NUSI': ('QQQ', 78, 0.63, 0.466, 0.164, -0.03, 0.39, 0.95, -0.2, -1.65), 'XYLD': ('SPY', 156, 0.641, 0.711, -0.07, -0.22, 0.1, 0.2, -0.35, -2.14), 'JEPI': ('SPY', 73, 0.592, 0.571, 0.021, -0.16, 0.2, 0.6, -0.14, -0.72), 'PBP': ('SPY', 222, 0.554, 0.614, -0.06, -0.19, 0.09, 0.21, -0.37, -3.69), 'DIVO': ('SPY', 114, 0.772, 0.729, 0.042, -0.1, 0.19, 0.73, 0.01, 0.07), 'SPYI': ('SPY', 46, 0.745, 0.72, 0.026, -0.05, 0.16, 0.71, -0.36, -5.49), 'SCHD': ('SPY', 176, 0.86, 0.839, 0.02, -0.18, 0.24, 0.56, -0.09, -0.69), 'VYM': ('SPY', 229, 0.865, 0.873, -0.008, -0.14, 0.13, 0.46, -0.1, -1.23), 'DVY': ('SPY', 229, 0.8, 0.819, -0.018, -0.24, 0.21, 0.46, -0.21, -1.39), 'SPHD': ('SPY', 164, 0.713, 0.754, -0.041, -0.34, 0.27, 0.39, -0.28, -1.16), 'NOBL': ('SPY', 152, 0.822, 0.894, -0.072, -0.28, 0.13, 0.24, -0.15, -0.85), 'RYLD': ('IWM', 86, 0.565, 0.593, -0.028, -0.21, 0.2, 0.43, -0.53, -3.15)}
print(f"{'fund':6}{'bench':6}{'n':>5}{'up':>8}{'down':>8}{'spread':>9}{'95% CI':>18}{'conv t':>9}")
for f, (b, n, up, dn, sp, lo, hi, p, cv, t) in R_SCORE.items():
    ci = f'[{lo:+.2f}, {hi:+.2f}]'
    print(f"{f:6}{b:6}{n:5d}{up:8.3f}{dn:8.3f}{sp:+9.3f}{ci:>18}{t:+9.2f}")


fund  bench     n      up    down   spread            95% CI   conv t
QYLD  QQQ     150   0.497   0.559   -0.062    [-0.21, +0.11]    -2.62
JEPQ  QQQ      49   0.712   0.689   +0.023    [-0.13, +0.24]    -5.58
NUSI  QQQ      78   0.630   0.466   +0.164    [-0.03, +0.39]    -1.65
XYLD  SPY     156   0.641   0.711   -0.070    [-0.22, +0.10]    -2.14
JEPI  SPY      73   0.592   0.571   +0.021    [-0.16, +0.20]    -0.72
PBP   SPY     222   0.554   0.614   -0.060    [-0.19, +0.09]    -3.69
DIVO  SPY     114   0.772   0.729   +0.042    [-0.10, +0.19]    +0.07
SPYI  SPY      46   0.745   0.720   +0.026    [-0.05, +0.16]    -5.49
SCHD  SPY     176   0.860   0.839   +0.020    [-0.18, +0.24]    -0.69
VYM   SPY     229   0.865   0.873   -0.008    [-0.14, +0.13]    -1.23
DVY   SPY     229   0.800   0.819   -0.018    [-0.24, +0.21]    -1.39
SPHD  SPY     164   0.713   0.754   -0.041    [-0.34, +0.27]    -1.16
NOBL  SPY     152   0.822   0.894   -0.072    [-0.28, +0.13]    -0.85
RYLD  IWM      86   

## 3. The two numbers are the same number

Look down the `up` and `down` columns: they track each other almost perfectly. QYLD keeps 50% of the good months and takes 56% of the bad ones. SCHD: 86% and 84%. VYM: 87% and 87%. The median spread across the whole panel is **-0.013** — that is, nothing.

And the confidence intervals swallow zero for **14 of 14** funds. Not one fund has a spread you could bet on.

> 🔬 **For the quants** — the spread's regression twin is the Henriksson-Merton convexity coefficient `b_up − b_dn`, which has an honest standard error. Zero of fourteen funds reach a nominal |*t*| ≥ 2 on the positive side. Six reach it on the **negative** side (QYLD, JEPQ, XYLD, PBP, SPYI, RYLD) — every one of them an option-writer, delivering exactly the concave payoff it sold.

## 4. The ruler is bent — it reads positive on nothing at all

Here is the part that changes how you read every fund fact sheet you will ever see. We built synthetic funds that are, by construction, **plain fractions of the index** — no cleverness, no options, true capture spread exactly zero — and measured them with the industry's own formula.

*(The cell below is synthetic by construction — it is a test of the measuring tape, not of any real fund.)*

In [2]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
from capture_ratio import data, strategy as st
nb = st.null_spread_distribution(
    data.synthetic_panel(signal_strength=0.0, seed=948 + s) for s in range(6))
print(f"{nb['n_obs']} synthetic funds, every one a plain fraction of the index")
print(f"true capture spread : 0.000 for all of them")
print(f"MEASURED spread     : mean {nb['spread_mean']:+.3f}, "
      f"positive {nb['spread_frac_positive']:.0%} of the time")
print(f"regression twin     : mean {nb['convexity_mean']:+.3f} (honest)")


42 synthetic funds, every one a plain fraction of the index
true capture spread : 0.000 for all of them
MEASURED spread     : mean +0.105, positive 81% of the time
regression twin     : mean -0.008 (honest)


The capture spread is a *ratio of two averages*, and the bottom of the down-capture ratio is a small negative number. Divide by something small and noisy and the answer wanders — upwards, as it happens. On data with a true spread of **exactly zero** the industry formula reads positive about **82%** of the time, with an average of **+0.111**.

**But we are not allowed to just subtract that from the real table**, and it is worth saying why, because it is the kind of shortcut that turns a true finding into an overstated one. The +0.111 averages over *many possible market histories*. The real funds were measured against **one** history — the actual one — and once you hold that fixed, most of the wobble disappears.

So we redid it properly: rebuild each real fund a thousand times as a **plain scaled index with no cleverness in it** — its own beta, its own noise, the real market's real months — and ask how often that fake fund's spread beats the real fund's. Answer: **0 of 14** real funds stand out (the closest call is *p* = 0.127), and the typical real fund reads **-0.035** *below* its own dumb twin. Smaller than the naive correction would have claimed — and still pointing the same way.

## 5. NUSI — the one fund that looked convex, and the split that did it

Before any of the above, one number had to be thrown out. On **2025-02-18** NUSI's price doubled in a single day — an exact ×2.000000, with QQQ unmoved. That is not a return, it is a **1-for-2 reverse split** that the data vendor's 'adjusted' prices had failed to back out, and it was worth a fake **+100% month**.

Left in, it made NUSI look like the one fund that works: down-capture **-0.309** (negative!), spread **+0.939**, alpha **+185.6 bps/month**. Repaired: down-capture **+0.466**, spread **+0.164**, alpha **+33.6 bps/month**. It also moved the panel headline — the confidence interval covers zero for 14/14 funds now, rather than 13/14.

NUSI still has the panel's largest spread at **+0.164**, and it still is not evidence: the interval [-0.03, +0.39] covers zero, and its regression twin is **-0.20** — *concave*, like every other option-writer.

## 6. Does last decade's scorecard predict the next one?

No. Split the sample at 2021 and rank the nine funds with enough history on both sides: the rank correlation between the two eras is **-0.117** (*p* = 0.76) and the spread keeps the *same sign* in only **44%** of cases — worse than a coin. Five of the nine funds flip outright. Whatever a capture spread describes, it is a sample, not a fund.

In [3]:
R_ERA = {'QYLD': (-0.107, -0.016), 'XYLD': (-0.193, 0.052), 'PBP': (-0.112, 0.073), 'DIVO': (0.017, 0.057), 'SCHD': (0.025, 0.002), 'VYM': (-0.037, 0.092), 'DVY': (-0.025, 0.112), 'SPHD': (-0.116, 0.034), 'NOBL': (-0.024, -0.125)}
RHO, PVAL, NOBS = (-0.117, 0.756, 9)
print(f"{'fund':6}{'2007-2020':>12}{'2021-2026':>12}   flip?")
for f, (a, b) in R_ERA.items():
    print(f"{f:6}{a:+12.3f}{b:+12.3f}   {'yes' if a * b < 0 else 'no'}")
print(f"\nSpearman rho between the two eras: {RHO:+.3f} "
      f"(p = {PVAL:.2f}, n = {NOBS})")


fund     2007-2020   2021-2026   flip?
QYLD        -0.107      -0.016   no
XYLD        -0.193      +0.052   yes
PBP         -0.112      +0.073   yes
DIVO        +0.017      +0.057   no
SCHD        +0.025      +0.002   no
VYM         -0.037      +0.092   yes
DVY         -0.025      +0.112   yes
SPHD        -0.116      +0.034   yes
NOBL        -0.024      -0.125   no

Spearman rho between the two eras: -0.117 (p = 0.76, n = 9)


## 7. So what *are* you buying?

A **smaller position in the index, plus a distribution schedule**. Every fund's beta sits between **0.52 and 0.87**, and not one of them has measurable alpha — the largest |*t*| on alpha across all fourteen is **1.56**, and it is *negative*. **1 of 14** beats its own benchmark on risk-adjusted return after subtracting cash — NUSI, and it wins on lower volatility, not higher return (its mean monthly excess return is *below* QQQ's).

The reduced drawdowns are real — eleven of fourteen funds cut the worst loss. But so does owning less of the index, and that costs 0.03% a year instead of 0.35–0.68%.

## Verdict

- **Signal — None.** Not one of the fourteen funds shows a positive capture spread at even a nominal |*t*| ≥ 2, never mind the **2.91** bar that fourteen simultaneous tests demand. The interval covers zero for 14/14, there is no rank persistence across eras (rho = -0.117), and against its own zero-cleverness twin not one fund stands out (smallest *p* = 0.127). The only robust finding runs the other way — six option-writers are significantly **concave**, which is what selling calls does by definition. Survivorship flatters the table: the funds that closed are gone from the tape.
- **Tradability — Mirage.** Nothing to buy and nothing to short. The beta-neutral spread that would monetise the scorecard earns **+0.8 bps/month gross** at its very best (*t* = +0.04) and is negative once the short leg pays 50 bps of borrow.
- **The honest reading.** These products are fairly priced beta reduction with a cash-flow calendar attached. That is a legitimate thing to want. It is simply not the asymmetry the sentence on the fact sheet describes.